docs/getting_started/install.md

```diff
@@ -29,10 +37,20 @@ kubectl -n argocd apply -f https://raw.githubusercontent.com/argoproj/argo-cd/st
 ```
 
 This will install the current stable version of Argo CD into the `argocd`
-namespace of your Kubernetes cluster.
+namespace of your Kubernetes cluster. Then continue reading [the first steps](getting_started/first_steps/).
+
+## Production-Grade Installation
 
-If you are planning a production-grade installation, please skip this step and
-read further.
+When you are ready to use Argo CD in a production environment you have
+several installation options.
+
+For a production setup you should pay attention to extra aspects of your installation such as:
+
+* defining the correct resource limits
+* applying proper security constraints
+* setting up high availability
+* managing Argo CD upgrades
+* monitoring Argo CD itself 
```

Comment:
Going forward, we should probably fill this with links to the respective sections in the docs

Reply:
Agreed